wordpress-mobile/WordPress-iOS
# Importing FormatableContent objects from WordPressKit

Fixes #9731 

This PR moves all `FormattableContent` code the main WP app.
As expected, Notifications didn't notice the change and it continues to work as before.

For Activity log, it was necessary a small change. There's a new wrapper class `FormattableActivity` that parse the activity `content` property as a `FormattableContentGroup`.

To test:
- Build and run.
- Check that all notifications continue to display as they should.
- Check that Activity Log formats its content as it should (the proper formatting will be applied later on)
